### PR TITLE
Add Cloud Config Client sample.

### PR DESCRIPTION
**Pull request by OlgaMaciaszek**

### CODE REVIEW

File: cloud-config-client/src/main/java/com/example/cloud/config/client/ConfigClientApplication.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		return Mono.just("test");
+	}
+
+}
+
+class ConfigClientApplicationHints implements RuntimeHintsRegistrar {
+
+	@Override
+	public void registerHints(RuntimeHints hints, ClassLoader classLoader) {
+		hints.reflection().registerType(TypeReference.of(ConfigClientApplication.class),
```
Comment (mhalbritter) [inline cloud-config-client/src/main/java/com/example/cloud/config/client/ConfigClientApplication.java:36]: Why is this necessary? Those hints should have been inferred from boot / SF.

Comment (OlgaMaciaszek) [reply cloud-config-client/src/main/java/com/example/cloud/config/client/ConfigClientApplication.java:36]: Righ @mhalbritter, it's unnecessary now. Will remove it.

Comment (OlgaMaciaszek) [reply cloud-config-client/src/main/java/com/example/cloud/config/client/ConfigClientApplication.java]: Done.

**Comment by mhalbritter** [comment]
Thank you!
